From f885c0cab12e24b4368f62603022d07a24d5a9ed Mon Sep 17 00:00:00 2001
From: Graham Moore <grmoore@opensource.altera.com>
Date: Fri, 27 Jan 2017 06:20:50 -0600
Subject: [PATCH 022/151] FogBugz #284482-2: A10 cache coherency is different
 than A5/C5.

commit  96d615ee73c299a1c84a7655a6446be9b9f94dc9 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

arm: dts: This patch adds the necessary devicetree entries to address
the differences in A5/C5 and A10 related to how the PL330 works.
A5/C5 requires DMA microcode to reside in uncached memory and
A10 requires DMA microcode to reside in cached memory.

Signed-off-by: Vince Bridgers <vbridger@opensource.altera.com>
Signed-off-by: Graham Moore <grmoore@opensource.altera.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/socfpga_arria10.dtsi b/arch/arm/boot/dts/socfpga_arria10.dtsi
index 6378e6461bf7..32561ab81f1f 100644
--- a/arch/arm/boot/dts/socfpga_arria10.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10.dtsi
@@ -68,6 +68,7 @@
 				#dma-requests = <32>;
 				clocks = <&l4_main_clk>;
 				clock-names = "apb_pclk";
+				microcode-cached;
 			};
 		};
 
-- 
2.26.1

